lib/views/manageofficerspage.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:parkr/views/registerofficerdialog.dart';
+
+class ManageOfficersPage extends StatefulWidget {
+  const ManageOfficersPage({Key? key}) : super(key: key);
+
+  @override
+  State<ManageOfficersPage> createState() => _ManageOfficersPageState();
+}
+
+class _ManageOfficersPageState extends State<ManageOfficersPage> {
+  List<Widget> _fetchOfficers() {
+    // Place holder widget
+    return <Widget>[
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack"),
+      const Text("ballsack")
```

Comment:
:eyes:

Reply:
@StephenCole19 these are definitely valid parking officers from the DB

Reply:
@justend29 I believe you, continue on 🥇